### ISSUE TITLE
classpath and Runtime.class

### ISSUE THREAD

**Issue by enedil** [tosum/instant_tests#1]
Hey,

how do you resolve the issue of classpath for JVM version? If I understand correctly, separate `Runtime.class` can be provided along with source code in `lib/` subdirectory, however you seem to assume that there is no additional runtime (the classpath `workdir` indicates that).

**Comment by tosum** [comment]
Yeah there isn't any support for separate `Runtime.class` :(, I don't have it in my solution.

I guess it would require adding `--class-path lib` to all places like
```
os.system(f"java --class-path workdir {class_name} > workdir/class.out")
```

Feel free to submit a PR that adds support for it
